sdk/src/main/java/io/radar/sdk/model/RadarConfig.kt

```diff
@@ -0,0 +1,22 @@
+package io.radar.sdk.model
+
+import org.json.JSONObject
+
+/**
+ * Remote configuration object
+ */
+internal data class RadarConfig(
+    val meta: RadarMeta,
+    val settings: RadarFeatureSettings
+) {
+
+    companion object {
+        private const val META = "meta"
+        private const val SETTINGS = "settings"
```

Comment:
`settings` is a bit vague here and I can see it getting unwieldy if people take liberties to stuff random things in there - personal preference for `featureSettings` or `featureFlags` to make it more descriptive

Reply:
Done. I used "featureSettings", since IMO "featureFlags" should be specifically for boolean values, whereas this contains other variable types.